backend/user/notifications.py

```diff
@@ -44,10 +44,11 @@ def send_shadow_push_notif_batch(tokens, body, isDev=False):
 def get_client(isDev):
     """Creates and returns APNsClient based on iOS credentials"""
 
-    auth_key_path = os.path.join(
-        os.path.dirname(os.path.dirname(os.path.abspath(__file__))), "ios_key.p8"
+    auth_key_path = os.environ.get(
+        "IOS_KEY_PATH",  # for dev purposes
```

Comment:
Even if we are using client for non-dev purposes, will this still work?

Reply:
If the env variable is not set, then it will default to the path laid out, so it should work just like before.